tests/std/tests/P0019R8_atomic_ref/test.cpp

```diff
@@ -269,6 +269,26 @@ void test_ptr_ops() {
     assert(ry.load() == a + 0x10);
 }
 
+// GH-1497 <atomic>: atomic_ref<const T> fails to compile
+void test_gh_1497() {
+    {
+        const int ci{1729};
+        const std::atomic_ref atom{ci};
+        assert(atom.load() == 1729);

```

Comment:
This doesn't test `store`, `exchange`, `compare_exchange_meow`, `operator=`, etc. which the Standard nonsensically requires to work ;)

Reply:
This will be covered by the LWG issue.